### PR TITLE
Pat analysis 7049

### PR DESCRIPTION
**Pull request by nabuelrub**
This code maps patients' record_id to mrn
https://bmi-work.kumc.edu/work/ticket/7049#ticket

### CODE REVIEW

File: cdm_cut.sql [hunk 1]
```diff
@@ -0,0 +1,166 @@
+/**
+ * CDM cut for DS-DETERMINED cohort
+ */
+DROP TABLE condition;
```
Comment (lav-patel) [inline cdm_cut.sql]: `DROP TABLE if exists condition;` will make your code idempotent, and same thing for the rest of the drop statment

File: cdm_cut.sql [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+
+CREATE TABLE condition AS
+SELECT
+    c.*
+FROM
+    cdm60_deid_dataset."condition" c
+    JOIN pcornet_trial pt ON pt.patid = c.patid;
```
Comment (lav-patel) [inline cdm_cut.sql]: Typically pcornet trial table has patients from multiple trails, You may want to pick only ds trail patients

select distinct trialid  from cdm60_limited_dataset.pcornet_trial pt ;
/*
ADPT
PREVENTABLE
PS_CMSLINKAGE_PILOT
*/

File: cdm_cut.sql (hunk 1 above)
Comment (nabuelrub) [reply cdm_cut.sql]: https://github.com/kumc-bmi/ds-determined-tools/commit/6291b8b5f72259e80e25415085a42f87b9534399

File: cdm_cut.sql (hunk 2 above)
Comment (nabuelrub) [reply cdm_cut.sql]: https://github.com/kumc-bmi/ds-determined-tools/commit/6291b8b5f72259e80e25415085a42f87b9534399

**Comment by nabuelrub** [comment]
I addressed all Lav's comments.

**Review by lav-patel — APPROVED** [review]

**Pull request merged**